app/models/taxon_presenter.js

```diff
@@ -0,0 +1,75 @@
+"use strict";
+
+var _ = require('lodash');
+var urlHelper = require('url');
+
+var Taxon = require('./taxon.js');
+var BreadcrumbMaker = require('../../lib/js/breadcrumb_maker.js');
+
+class TaxonPresenter {
+  constructor (taxonParam, taxonomyData) {
+    this.taxonomyData = taxonomyData;
+    this.basePath = "/alpha-taxonomy/" + taxonParam;
+
+    this.build();
+  }
+
+  build () {
+    this.buildTaxon();
+    this.buildBreadcrumb();
+    this.buildParent();
+    this.buildChildren();
+    this.buildContent();
+    this.checkIfPenultimate();
+  }
+
+  buildTaxon () {
+    this.taxon = Taxon.fromMetadata(this.basePath, this.taxonomyData);
+    this.title = this.taxon.title;
+    this.description = this.taxon.description;
+  }
+
+  buildBreadcrumb () {
+    var breadcrumbMaker = new BreadcrumbMaker(this.taxonomyData);
+    this.breadcrumb =  breadcrumbMaker.getBreadcrumbForTaxon([this.basePath]);
+  }
+
+  buildParent () {
+    this.parent = this.breadcrumb[this.breadcrumb.length - 1];
+  }
+
+  buildChildren () {
+    this.children = this.taxon.atozChildren();
+    for (var i = 0; i < this.children.length; i += 1) {
```

Comment:
An alternative to this iteration, you could use the [`for...of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) notation here, e.g.:

```
for(const child of this.children) {
   child. ....
}
```